### PR TITLE
Include PR/Issue links in the changelog

### LINKED ISSUE #1627: Chloggen should generate links for PRs
Changelog contains PR numbers which are not automatically recognized as links when they are committed to the repo.

https://github.com/open-telemetry/semantic-conventions/blob/07293edd52e2182f9c18e871bbff44958cf77271/CHANGELOG.md?plain=1#L15

It'd be great to make chloggen produce links for them.

### PR DESCRIPTION
**Pull request by joaopgrassi**
Fixes #1627

## Changes

Adds a template for customizing the generated changelog file by `chloggen`. The template is a copy of the default, but renders links for issues/PR numbers specified in the `issues` field in the changelog entries.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: .chloggen/CHANGELOG.tmpl
Comment (trask) [inline .chloggen/CHANGELOG.tmpl:1]: is there a "default template" somewhere that you used, or did you have to write from scratch? thanks!

Comment (jsuereth) [reply .chloggen/CHANGELOG.tmpl:1]: Can you add a link to the default template so we can track changes going forward?

**Review by jsuereth — APPROVED** [review]

File: .chloggen/CHANGELOG.tmpl
Comment (joaopgrassi) [reply .chloggen/CHANGELOG.tmpl:1]: Done: [41267bf](https://github.com/open-telemetry/semantic-conventions/pull/1630/commits/41267bfb6b2edc6b4c122e22ebeb12ac3f5a3c6d)

**Review by trask — APPROVED** [review]

**Review by lmolkova — APPROVED** [review]
Thank you!

**Comment by lmolkova** [comment]
@joaopgrassi I'd merge, but I can't update your branch :(

image

**Pull request merged**